src/Upgrade/Infrastructure/Report/Dto/ReportMetadataDto.php

```diff
@@ -41,28 +43,44 @@ class ReportMetadataDto
      */
     protected string $reportId;
 
+    /**
+     * @var int
+     */
+    private int $idRg;
```

Comment:
I prefer to rename it into rgId

Reply:
we write idGr for singular and rgIds for plural